homeassistant/components/modem_callerid/services.yaml

```diff
@@ -0,0 +1,6 @@
+reject_call:
+  description: Reject incoming call.
+  fields:
```

Comment:
Does it make sense to use `target:` here instead?

Reply:
It's an entity service so that works for me.